docs/playbooks/greenhouse.md

```diff
@@ -0,0 +1,136 @@
+# Greenhouse Playbook
```

Comment:
Thoughts on making this show up as a move rather than a new file?

Reply:
The addition of tombstones in the old locations prevents git from seeing this as a move when looking at the entire branch

I'm going to keep the addition of tombstones as the last commit, and you can use the following compare link to see what's actually changed: https://github.com/kubernetes/test-infra/compare/master...spiffxp:doc-prow-monitoring%5E

Or look at just the first commit to see the move